pipelines/concourse.yml

```diff
@@ -786,6 +787,39 @@ jobs:
   on_success: *fixed-concourse
   on_failure: *broke-concourse
 
+- name: bin-smoke-containerd
+  public: true
+  serial: true
+  plan:
+    - in_parallel:
+        - get: concourse
+          passed: [build-rc]
+          trigger: true
+        - get: version
+          passed: [build-rc]
+          trigger: true
+        - get: linux-rc-ubuntu
+          passed: [build-rc]
+          trigger: true
+        - get: unit-image
+        - get: ci
+    - task: terraform-smoke
+      image: unit-image
+      file: ci/tasks/terraform-smoke.yml
+      input_mapping: {linux-rc: linux-rc-ubuntu}
+      params:
+        GCP_PROJECT: cf-concourse-production
+        GCP_KEY: ((concourse_smoke_gcp_key))
+        SSH_KEY: ((concourse_smoke_ssh_key))
+        DEPLOYMENT: smoke
+        USE_CONTAINERD: true
```

Comment:
```suggestion
        USE_CONTAINERD: true
        WORKSPACE: containerd
```

Reply:
where does `containerd` got consumed? But I got your point, that is cleaner than using -backend-config. I will try that thx!

Reply:
NVM. I saw the script is already consuming the env var. Neat!